stl/inc/chrono

```diff
@@ -5136,54 +5136,95 @@ namespace chrono {
         return _Istr;
     }
 
+    template <class _CharT, class _Traits, class _Alloc, class _Parsable>
+    struct _Time_parse_iomanip_c_str {
+        _Time_parse_iomanip_c_str(const _CharT* _Fmt_, _Parsable& _Tp_,
+            basic_string<_CharT, _Traits, _Alloc>* _Abbrev_ = nullptr, minutes* _Offset_ = nullptr)
+            : _Fmt{_Fmt_}, _Tp{_Tp_}, _Abbrev{_Abbrev_}, _Offset{_Offset_} {}
+        _Time_parse_iomanip_c_str(_Time_parse_iomanip_c_str&&) = delete;

```

Comment:
I don't know how others feel, but I'd prefer a space here. Same for `_Time_parse_iomanip` below.
```suggestion
            : _Fmt{_Fmt_}, _Tp{_Tp_}, _Abbrev{_Abbrev_}, _Offset{_Offset_} {}
            
        _Time_parse_iomanip_c_str(_Time_parse_iomanip_c_str&&) = delete;
```